tests/aws/services/s3/test_s3_list_operations.py

```diff
@@ -39,6 +40,18 @@ def _endpoint_url(region: str = "", localstack_host: str = None) -> str:
     return config.internal_service_url(host=f"s3.{region}.{LOCALHOST_HOSTNAME}")
 
 
+def assert_timestamp_is_iso8061_s3_format(timestamp: str):
+    # the timestamp should be looking like the following
+    # 2023-11-15T12:02:40.000Z
+    assert timestamp.endswith(".000Z")
+    assert len(timestamp) == 24
+    # assert that it follows the right format and it does not raise an exception during parsing
+    parsed_ts = datetime.datetime.strptime(timestamp, "%Y-%m-%dT%H:%M:%S.%fZ")
+    assert parsed_ts.microsecond == 0
+
+    return True
```

Comment:
nit: It's not really necessary to `return True` here and only use it with `assert` statements. If any of the asserts here is failing, then an exception is thrown anyway. If not, the return value is always `True`.

Reply:
yes I agree, but I used `assert` under just for the look of validating something in the test 😬 it felt weird to validate the test with a simple method call. But it might make it a bit more... verbose and weird, I agree.

Maybe it's the same debate as `snapshot.match` 😅

Reply:
The return value would be `None` no?

Reply:
Yes, the return value would be `None`, but I would just go with the assert method call without an assert statement in the test itself. The method is clearly named, the additional sugar is confusing and can never be triggered anyways.

Reply:
Will update, I'm convinced 😉